tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -145,10 +380,13 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
                 tokens.append("[SEP]" if self.support_vocab else SEP_ID)
                 segment_ids.append(0)
 
-                for i in range(doc_span.length):
-                    split_token_index = doc_span.start + i
-                    tokens.append(all_doc_tokens[split_token_index])
-                    segment_ids.append(1)
+                try:
+                    for i in range(doc_span.length):
+                        split_token_index = doc_span.start + i
+                        tokens.append(all_doc_tokens[split_token_index])
+                        segment_ids.append(1)
+                except TypeError as e:
```

Comment:
@mzuevx do you know, in which scenarious here TypeError possible?

Reply:
I think that exception handling code can be removed